packages/esbuild/esbuild.bzl

```
@@ -168,9 +187,19 @@ See https://esbuild.github.io/api/#define for more details
             doc = "A list of direct dependencies that are required to build the bundle",
         ),
         "entry_point": attr.label(
-            mandatory = True,
             allow_single_file = True,
-            doc = "The bundle's entry point (e.g. your main.js or app.js or index.js)",
+            doc = """The bundle's entry point (e.g. your main.js or app.js or index.js)
+
+This is just a shortcut for the `entry_points` attribute with a single entry.
+Either this attribute or `entry_point` must be specified, but not both.
```

I think this is copy / paste from rollup, but nit, I don't really like the word "just" 😛 

```suggestion
This is a shortcut for the `entry_points` attribute with a single entry.
Specify either this attribute or `entry_point`, but not both.
```